ml/README.md

```diff
@@ -0,0 +1,249 @@
+
+### Anomaly Bit - (100 = Anomalous, 0 = Normal)
+
+Once each model is trained, Netdata will begin producing an ["anomaly score"](#anomaly-score) at each timestep for each dimension. This ["anomaly score"](#anomaly-score) is essentially a distance measure to the trained cluster centers of the model (by default each model has k=2, so two cluster centers are learned). More anomalous looking data should be more distant to those cluster centers. If this ["anomaly score"](#anomaly-score) is sufficiently large this is a sign that the recent raw values of the dimension could potentially be anomalous. By default, "sufficiently large" means that the distance is in the 99th percentile or above of all distances observed during training or, put another way, it has to be further away than the furthest 1% of the data used during training. Once this threshold is passed the ["anomaly bit"](#anomaly-bit) corresponding to that dimension is set to 100 to flag it as anomalous, otherwise it would be left at 0 to signal normal data.
+
+What this means is that in addition to the raw value of each metric, Netdata now also would have available an ["anomaly bit"](#anomaly-bit) that is either 100 (anomalous) or 0 (normal). Importantly, this is achieved without additional storage overhead due to how the anomaly bit has been implemented within the existing internal Netdata storage representation.
+
+This ["anomaly bit"](#anomaly-bit) is exposed via the `anomaly-bit` key that can be passed to the `options` param of the `/api/v1/data` REST API. 
+
+For example, here are some recent raw dimension values for `system.ip` on our [london](http://london.my-netdata.io/) demo server:
+
+[`https://london.my-netdata.io/api/v1/data?chart=system.ip`](https://london.my-netdata.io/api/v1/data?chart=system.ip)
+
+```
+{
+ "labels": ["time", "received", "sent"],
+    "data":
+ [
+      [ 1638365672, 54.84098, -76.70201],
+      [ 1638365671, 124.4328, -309.7543],
+      [ 1638365670, 123.73152, -167.9056],
+      ...
+ ]
+}
+```
+
+And if we add the `&options=anomaly-bit` params, we can see the "anomaly bit" value corresponding to each raw dimension value: 
+
+[`https://london.my-netdata.io/api/v1/data?chart=system.ip&options=anomaly-bit`](https://london.my-netdata.io/api/v1/data?chart=system.ip&options=anomaly-bit)
+
+```
+{
+ "labels": ["time", "received", "sent"],
+    "data":
+ [
+      [ 1638365672, 0, 0],
+      [ 1638365671, 0, 0],
+      [ 1638365670, 0, 0],
+      ...
+ ]
+}
+```
+
+Typically, the anomaly bit will mostly be 0 under normal circumstances with some random fluctuations every now and then to 100. Although this very much depends on the nature of the dimension in question.
+
+### Anomaly Rate - avg(anomaly bit)
```

Comment:
Is there a specific reason to use 'avg' instead of average?

Reply:
nope, updated